e2e/web/specs/releasenotes.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test } from "../fixture";
+import { HeaderComponent } from "../components/header.component";
+import { ReleaseNotesPage } from "../page/releasenotes.page";
+import json from "../../../Changelog.json";
+import { ReleaseNoteInterface } from "../../../app/Components/ReleaseNotes/ReleaseNote";
+const releaseNotes: ReleaseNoteInterface[] = json;
+
+test.describe("release notes", () => {
```

Comment:
Can you add two more tests, one to verify that feature and preview list contain correct contents when available and one to verify they contain None otherwise?

Reply:
Might be a bit tricky, I'll have to figure out how to get playwright to scroll down to a specific release note to test those instances. (If we had component-level testing this is where I think it would be better as a component test instead of end to end test) But I agree it is a good test to have, will work on that.

Reply:
I found a method with playwright, may make this easier than I thought it would be:
https://playwright.dev/docs/api/class-locator#locator-scroll-into-view-if-needed

Reply:
didn't think about that, hopefully test doesn't take too long to scroll to bottom

Reply:
I've updated the tests